src/mapgen_v6.cpp

```diff
@@ -163,7 +182,7 @@ void MapgenV6Params::readParams(const Settings *settings)
 
 void MapgenV6Params::writeParams(Settings *settings) const
 {
-	settings->setFlagStr("mgv6_spflags", spflags, flagdesc_mapgen_v6, (u32)-1);
+	settings->setFlagStr("mgv6_spflags", spflags, flagdesc_mapgen_v6, (u32) - 1);
```

Comment:
Why add spaces around the negative sign here? The intent is to cast the signed integer value of -1 to a 32 bit unsigned value (thus making a 32 bit integer data type filled with set bits)